includes/modules/payment/securesubmit.php

```diff
@@ -219,7 +221,19 @@ public function before_process() {
 
         return false;
     }
-
+    
+    protected function getConfig()
+    {
+        $config = new ServicesConfig();
+        $config->secretApiKey = MODULE_PAYMENT_SECURESUBMIT_SECRET_API_KEY;
+        $config->serviceUrl = ($this->enableCryptoUrl) ?
+                              'https://cert.api2-c.heartlandportico.com/':
+                              'https://cert.api2.heartlandportico.com';
```

Comment:
This method needs to handle the different URLs for production and sandbox/certification.

Reply:
updated